### PR TITLE
Converter: Rename 3.x Vector2 `clamped` to `limit_length`

### PR DESCRIPTION
**Pull request by tlobig**
Doc already said for 3.5 clamped is deprecated and limit_lenght should be used, clamp is the wrong function

### CODE REVIEW

File: editor/renames_map_3_to_4.cpp [hunk 1]
```diff
@@ -579,7 +579,7 @@ const char *RenamesMap3To4::gdscript_function_renames[][2] = {
 	// Remember to add them to the builtin_types_excluded_functions variable, because for now these functions cannot be listed.
 	//	{ "empty", "is_empty" }, // Array -- Used as custom rule. Be careful, this will be used everywhere.
 	//	{ "remove", "remove_at" }, // Array -- Breaks Directory and several more.
-	{ "clamped", "clamp" }, // Vector2 -- Be careful, this will be used everywhere.
+	{ "clamped", "limit_length" }, // Vector2 -- Doc already said for 3.5 clamped is deprecated and limit_lenght should be used
```
Comment (akien-mga) [inline editor/renames_map_3_to_4.cpp:582]: The comment (and the commit message) has a typo, but I'd simply remove it, it's the correct rename and doesn't require justification in the code.

Comment (tlobig) [reply editor/renames_map_3_to_4.cpp]: thank you for the input, I reverted the comment to what it was previously. I am 100% sure the rename is correct after this change, but would expect to get confirmation from anyone else

**Comment by akien-mga** [comment]
Looks good! Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by tlobig** [comment]
happy to oblidge

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
The CI failed due to a workaround that needed to be updated, so I amended the commit to fix it up, and also added the missing C# conversion.

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**